configure.ac

```diff
@@ -1590,6 +1590,14 @@ if test "x$enable_ipv6" = "xauto"; then
         fi
 fi
 
+AC_CHECK_DECLS([TCP_KEEPIDLE, TCP_KEEPINTVL, TCP_KEEPCNT], [], [],
+                [[#include <sys/socket.h>
+                  #include <netinet/in.h>
+                  #include <netinet/tcp.h>]])
+
+if test "x$ac_cv_have_decl_TCP_KEEPIDLE" = "xyes" -a "x$ac_cv_have_decl_TCP_KEEPINTVL" = "xyes" -a "x$ac_cv_have_decl_TCP_KEEPCNT" = "xyes"; then
```

Comment:
You could use the `action-if-found` for this case.

Reply:
`action-if-found` does not work as documented (autoconf v2.69).
I need all 3 symbols to present, my autoconf version runs `action-if-found` when any of the 3 are available.